### ISSUE TITLE
Paragon CLI `install-theme` doesn't reset back to original `@edx/brand` after usage

### ISSUE THREAD

**Issue by adamstankiewicz** [openedx/paragon#2644]
## Improvements

The Paragon CLI tool's `install-theme` command works great to override the `@edx/brand` NPM alias during local development without risking modifying/committing code. Testing the command in the frontend-template-application MFE, it appears that after using `paragon install-theme`, the provided NPM package (e.g., `@edx/brand-edx.org@latest`) remains installed even when I revert back to using the default/standard `npm start` command instead of `npm run start:with-theme`.

**Testing instructions**
1. Install latest `@edx/paragon` into an MFE.
2. Add NPM script for `start:with-theme`, which runs `paragon install-theme && npm start`.
3. Execute `npm run start:with-theme` and provide `@edx/brand-edx.org@latest` as input.
4. Observe custom theme is applied in running application within browser.
5. Stop the dev server and start it again with `npm start` instead.
6. Observe the `@edx/brand-edx.org@latest` theme is still installed even though I would have expected the default Paragon theme to be applicable again.
7. As is, need to run `npm install` to reset back to original theme `@edx/brand-openedx`.

**Comment by adamstankiewicz** [comment]
Two possible approaches for the fix:

1. Automatically revert back to `@edx/brand-openedx` upon killing dev server.
2. Document (either in README and/or CLI tool itself) that `npm install` is necessary to reset to default `@edx/brand-openedx`.

**Comment by PKulkoRaccoonGang** [comment]
PR: https://github.com/openedx/paragon/pull/2768

**Comment by adamstankiewicz** [comment]
@PKulkoRaccoonGang I'm curious if you explored option 1 (i.e., "Automatically revert back to @edx/brand-openedx upon killing dev server.")?

**Comment by PKulkoRaccoonGang** [comment]
@adamstankiewicz Yes, I didn’t spend a lot of time researching this problem, but I tried running `npm uninstall ${packageName}` through `child_process` to remove the installed theme from `node_modules`, but it didn’t work.

**Comment by PKulkoRaccoonGang** [comment]
@viktorrusakov  suggested an implementation using the following command:`"start:with-theme": "paragon install-theme && npm start && npm install @edx/brand"`.
